src/traces/sunburst/attributes.js

```diff
@@ -215,5 +215,18 @@ module.exports = {
     },
     sort: pieAttrs.sort,
 
+    root: {
+        color: {
+            valType: 'color',
+            editType: 'style',
```

Comment:
In order to `Plotly.restyle(gd, 'root.color', 'green');` to work 
this `editType` should also be `calc` noting that this attribute has an impact on the `calc` step.

Reply:
addressed in 2f07963